### ISSUE TITLE
WAVE accessibility issues for RAMP editor

### ISSUE THREAD

**Issue by yileifeng** [ramp4-pcar4/storylines-editor#381]
## Description

Getting WAVE errors on the RAMP editor:
- missing form labels for all inputs
- empty buttons for all information mini-icons

## Steps to reproduce

1. Go to map editor on any product
2. Run WAVE and see errors popup
